baxter_collaboration_lib/test/test_utils_lib.cpp

```diff
@@ -89,6 +89,62 @@ TEST(UtilsLib, State)
 
 }
 
+// sarim's tests
+TEST(UtilsLib, toStringConversions)
+{
+    int testInt;
+    testInt = 10;
+    EXPECT_EQ("10", toString(testInt));
+
+    std::vector<int> testIntVector;
+    testIntVector.push_back(10);
+    testIntVector.push_back(20);
+    EXPECT_EQ("[10, 20]", toString(testIntVector)); 
+
+    double testDouble;
+    testDouble = 10.123;
+    EXPECT_EQ("10.123", toString(testDouble));
+
+    std::vector<double> testDoubleVector;
+    testDoubleVector.push_back(10.123);
+    testDoubleVector.push_back(20.123);
+    EXPECT_EQ("[10.123, 20.123]", toString(testDoubleVector));
+
+    geometry_msgs::Pose testPose;
+    testPose.position.x = 0; testPose.position.y = 5; testPose.position.z = 10;
+    testPose.orientation.x = 15; testPose.orientation.y = 20; testPose.orientation.z = 25; 
+    testPose.orientation.w = 30;
+    EXPECT_EQ("{position:{x: 0, y: 5, z: 10}, orientation:{x: 15, y: 20, z: 25, w: 30}}", toString(testPose));
+}
```

Comment:
Good job here! But try also to think about edge cases, eg negative numbers, zeros, things that might break the code.